src/libraries/Microsoft.Extensions.FileProviders.Physical/src/Microsoft.Extensions.FileProviders.Physical.csproj

```
@@ -14,6 +14,16 @@
              Link="Common\src\Extensions\NonCapturingTimer\NonCapturingTimer.cs" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetFramework)' != 'netstandard2.0' and !$(TargetFramework.StartsWith('net4'))">
```

Given that the files are named, `netstandard2.1` I think it would be better to condition this itemgroup to when `TargetFramework==netstandard2.1`.

```suggestion
  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.1'">
```

---

agree, thanks